### PR TITLE
Small refactor

### PR DESCRIPTION
**Pull request by IS2511**
## Proposed changes

Implementing some of the minor changes discussed in my review of PR #63 

- https://github.com/ScuffleTV/scuffle/pull/63#discussion_r1126619829
- https://github.com/ScuffleTV/scuffle/pull/63#discussion_r1126094642
- https://github.com/ScuffleTV/scuffle/pull/63#discussion_r1126636700
- https://github.com/ScuffleTV/scuffle/pull/63#discussion_r1126325355
- https://github.com/ScuffleTV/scuffle/pull/63#discussion_r1126338350

## Types of changes

What types of changes does your code introduce to Scuffle?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/ScuffleTV/scuffle/blob/main/CONTRIBUTING.md) doc
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

Time to sign that CLA :Clueless:

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ VITE_GQL_VERSION=1.0
 Then you can export the following environment variables:
 
 ```bash
-export SCUF_TURNSTILE_SECRET_KEY=<secret-key>
+export SCUF_TURNSTILE_DUMMY_SECRET=<secret-key>
```
Comment (TroyKomodo) [inline CONTRIBUTING.md:169]: No this is wrong, the env variable is `SCUF_TURNSTILE_SECRET_KEY`

File: backend/api/src/config.rs [hunk 2]
```diff
@@ -17,7 +17,7 @@ pub struct AppConfig {
     pub database_url: String,
 
     /// The Cloudflare Turnstile site key to use
-    pub turnstile_secret_key: String,
+    pub turnstile_dummy_secret: String,
```
Comment (TroyKomodo) [inline backend/api/src/config.rs:20]: No this is `turnstile_secret_key`

File: backend/api/src/config.rs [hunk 3]
```diff
@@ -36,7 +36,7 @@ impl Default for AppConfig {
             config_file: "config".to_string(),
             bind_address: "[::]:8080".to_string(),
             database_url: "postgres://postgres:postgres@localhost:5432/scuffle-dev".to_string(),
-            turnstile_secret_key: "6LcZ8uIUAAAAAAM8ZQXZ0Z0Z0Z0Z0Z0Z0Z0Z0Z0Z".to_string(),
+            turnstile_dummy_secret: "6LcZ8uIUAAAAAAM8ZQXZ0Z0Z0Z0Z0Z0Z0Z0Z0Z0Z".to_string(),
```
Comment (TroyKomodo) [inline backend/api/src/config.rs:39]: Change the VALUE to `"turnstile_dummy_secret"` not the key.

**Comment by TroyKomodo** [comment]
If you are going to rename the `extend_gql` method you might as well also rename the `extend_route` method too.

File: backend/api/src/config.rs (hunk 3 above)
Comment (IS2511) [reply backend/api/src/config.rs:39]: lmao

**Review by TroyKomodo — APPROVED** [review]
Lgtm please rebase and ill merge.

**Review by 4w3Official — APPROVED** [review]
LGTM

**Review by SputNikPlop — APPROVED** [review]
LGTM

**Pull request merged**